frontend/components/project/ProjectList.vue

```diff
@@ -87,7 +89,8 @@ export default Vue.extend({
     return {
       search: this.$route.query.q,
       options: {} as DataOptions,
-      mdiMagnify
+      mdiMagnify,
+      projects: this.items
```

Comment:
Is this `projects` variable used anywhere?

Reply:
No, I initially used it for a different approach but now it is not used. Sorry I missed removing this line.